addons/website_sale/static/src/scss/product_configurator.scss

```
@@ -1,31 +1,26 @@
 .css_attribute_color {
+    --css-attribute-color-border-width: .25rem;
+
     position: relative;
     display: inline-block;
-    border: 5px solid $input-border-color;
+    border: var(--css-attribute-color-border-width) solid $body-bg;
```

I'd use the CSS variable here, it would allow us to tweak it if necessary

```suggestion
    border: var(--css-attribute-color-border-width) solid var(--body-bg);
```